lib/blocks.php

```diff
@@ -592,3 +592,31 @@ function gutenberg_multiple_block_styles( $metadata ) {
 	return $metadata;
 }
 add_filter( 'block_type_metadata', 'gutenberg_multiple_block_styles' );
+
+/**
+ * Get theme styles variations.
+ *
+ * @return array
+ */
+function gutenberg_get_theme_styles_variations() {
```

Comment:
Can we move this to a lib/wordpress-6.0/blocks.php or something like that. (See https://github.com/WordPress/gutenberg/blob/trunk/lib/README.md)

Reply:
Also I feel blocks.php might not be the right place for this, maybe this could be a `WP_theme` method when it lands in Core?

Reply:
I guess these comments might not be relevant depending on whether we address https://github.com/WordPress/gutenberg/pull/39886#discussion_r838491541